### ISSUE TITLE
staticcheck -f binary fails

### ISSUE THREAD

**Issue by MaximMolchanov** [dominikh/go-tools#1370]
Runinng staticcheck with `-f binary` parameters fails:
```
$ go install honnef.co/go/tools/cmd/staticcheck@v0.4.1
$ staticcheck -f binary ./... > tmpfile
failed writing output: gob: type lintcmd.lintResult has no exported fields
```

go version go1.19.4 linux/amd64

**Comment by dominikh** [comment]
Fixed and released as version 2023.1.2.

**Comment by MaximMolchanov** [comment]
Wow, so fast response! Awesome!
